docs/getting-started/index.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Getting started with Apache Gravitino"
+slug: /getting-started
+license: "This software is licensed under the Apache License version 2."
+---
+
+There are several options for getting started with Apache Gravitino.
+
+<!--Docker option-->
+Installing and configuring Hive and Trino can be a little complex.
+If you are unfamiliar with the technologies, using Docker might be a good choice.
+There are pre-packaged containers for Gravitino, Apache Hive, Apache Hadoop,
+Trino, MySQL, PostgesSQL, and others,
+check [installing Gravitino playground](./playground.md) for more details.
+
+This page guides you through the process to download and install Gravitino
+from source.
+
+1. [Prepare environment](#environment-preparation)
+   - Deploy and run Gravitino on [Amazon Web Service (AWS)](#aws)
+   - Deploy and run Gravitino on [Google Compute Platform (GCP)](#gcp)
+   - Run a Gravitino on [your own machine](#local)
+1. [Install Gravitino](#install-gravitino)
+1. [Start Gravitino](#start-gravitino)
+1. [Install Apache Hive](#install-apache-hive)
+1. [Interact with Apache Gravitino API](#interact-with-apache-gravitino-api)
+
+> [!NOTE]
```

Comment:
We use Docusaurus framework, it supports extended syntax (https://docusaurus.io/docs/2.x/markdown-features/admonitions) for such usage, you can try it.

Reply:
I would suggest using Docusaurus syntax, which will be rendered by the framework and better displayed on the website.